apps/system-e2e/src/tests/judicial-system/smoke/search-warrant-tests.spec.ts

```diff
@@ -90,4 +91,59 @@ test.describe.serial('Search warrant tests', () => {
     await page.getByRole('button', { name: 'Loka glugga' }).click()
     await expect(page).toHaveURL('/krofur')
   })
+
+  test.describe.serial('Search warrant court tests', () => {
+    test('court should receive search warrant request and make a ruling', async ({
+      judgePage,
+    }) => {
+      const page = judgePage
+      const yesterday = subDays(new Date(), 1).toLocaleDateString('is-IS', {
+        month: '2-digit',
+        day: '2-digit',
+        year: 'numeric',
+      })
+
+      await page.goto(`domur/rannsoknarheimild/mottaka/${caseId}`)
+      await page.getByTestId('courtCaseNumber').click()
+      await page.getByTestId('courtCaseNumber').fill(randomCourtCaseNumber())
+      await page
+        .getByText('Veldu dómara/aðstoðarmann *Veldu héraðsdómara')
+        .click()
+      await page.locator('#react-select-judge-option-1').click()
+      await page.getByTestId('continueButton').click()
+      await expect(page).toHaveURL(`domur/rannsoknarheimild/yfirlit/${caseId}`)
+      await page.getByTestId('continueButton').click()
+      await page.getByText('Fulltrúar málsaðila boðaðir').click()
+      await page.getByTestId('continueButton').click()
+      await page.getByTestId('modalPrimaryButton').click()
+      await page.getByText('Krafa samþykkt').click()
+      await page.getByTestId('continueButton').click()
+      await page.getByLabel('Dagsetning þingfestingar *').click()
+      await page.locator('input[id=courtStartDate]').fill(yesterday)
+      await page.getByTestId('courtStartDate-time').click()
+      await page.getByTestId('courtStartDate-time').fill('12:00')
+      await page
+        .locator('label')
+        .filter({ hasText: 'Varnaraðili unir úrskurðinum' })
+        .click()
+      await page
+        .locator('label')
+        .filter({ hasText: 'Sækjandi tekur sér lögboðinn frest' })
+        .click()
+      await page.getByLabel('Dagsetning uppkvaðningar *').click()
+      await page.locator('input[id=courtEndTime]').fill(yesterday)
```

Comment:
Roles again. Not tested, only a suggestion
```suggestion
      await page.getByRole('date', {name: "Veldu lokadagsetningu"}).fill(yesterday)
```

Reply:
This input (react-datepicker from island-ui) doesn't have a role date set. There are also two date pickers on this page, so I'm going to leave this as is for now.